tests/tool/Integration.Tests/IntegrationScenarios/PCL/Upgraded/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is this still here? This should have been removed

Reply:
fixed